.github/workflows/testing.yml

```diff
@@ -15,10 +19,6 @@ jobs:
     - name: Checkout code
       uses: actions/checkout@v2
 
-    - name: Build
-      run: make build
-
```

Comment:
since deleted `bls.yml` might be good to test it compileable as well on other OSs so avoid facing compile issues on external lib

Reply:
All targets have BLS now.